src/features/emails/components/EmailEditor/EmailSettings/ButtonBlockListItem.tsx

```diff
@@ -39,7 +39,11 @@ const ButtonBlockListItem: FC<ButtonBlockLIstItemProps> = ({
       hasErrors={hasErrors}
       readOnly={readOnly}
       selected={selected}
-      title={messages.editor.tools.button.title()}
+      title={
+        data.buttonText
+          ? messages.editor.tools.button.title()
+          : messages.editor.tools.button.settings.defaultButtonTextWarning()
```

Comment:
I was thinking that maybe the error message could be moved out of the header? I mean the block is still a button even if it's invalid, so I think it might be a bit jarring to replace the title like this.

Compare this with how invalid links are handled in paragraph/text blocks. It's still a text block and titled as such ("Text"), but there's an additional warning underneath.

![image](https://github.com/user-attachments/assets/05a97924-f650-4c38-844f-03755d19e3ae)

Reply:
Thank you! Of course that would be a more suitable way to handle it. I'll be back with an updated version shortly.

Reply:
![several-button-text-errors](https://github.com/user-attachments/assets/be6f87bd-5a9c-4a25-a8e5-45bf8e1f7f75)

It seems to be doing what I intended for it to do: displays error when the button has its default text, when the button has zero characters and when the button texts consists only of whitespaces.